src/tracer.js

```diff
@@ -31,6 +30,14 @@ import BaggageSetter from './baggage/baggage_setter';
 import DefaultThrottler from './throttler/default_throttler';
 import uuidv4 from 'uuid/v4';
 
+let pjson;
+try {
+  pjson = require('../package.json');
+} catch (_) {
+  // $FlowIgnore
+  pjson = require('../../package.json');
```

Comment:
I think this is dangerous. If we do this, let's add an assert that the package name is actually jaeger.